ortools/sat/python/cp_model.py

```diff
@@ -682,6 +686,14 @@ def __repr__(self):
     def Name(self):
         return self.__ct.name
 
+    def Start(self):
```

Comment:
I would do: def Start(self, model):
  return IntVar(model, self.__ct.interval.start)

with a new IntVar ctor

Reply:
Ups, I misunderstood your earlier comment, I thought you disagreed with the whole idea. I will correct this later

Reply:
One thing that could be troublesome is that this will create a new object, so if someone expects the same object for things like dictionaries, sets and so on, it wont work. Unless we also redefine `__hash__` with the index or something